chpl/chpl-api/src/main/java/gov/healthit/chpl/web/controller/UserManagementController.java

```diff
@@ -96,24 +103,29 @@
                     + "the following: 1) /invite 2) /create or /authorize 3) /confirm ")
     @RequestMapping(value = "/create", method = RequestMethod.POST, consumes = MediaType.APPLICATION_JSON_VALUE,
     produces = "application/json; charset=utf-8")
-    public User createUserDeprecated(@RequestBody final CreateUserFromInvitationRequest userInfo)
-            throws InvalidArgumentsException, UserCreationException, UserRetrievalException, EntityRetrievalException,
-            MessagingException, JsonProcessingException, EntityCreationException {
+    public @ResponseBody User createUserDeprecated(@RequestBody final CreateUserFromInvitationRequest userInfo)
+            throws ValidationException, EntityRetrievalException, InvalidArgumentsException, UserRetrievalException,
+            UserCreationException, MessagingException, JsonProcessingException, EntityCreationException {
 
         return create(userInfo);
     }
 
     private User create(final CreateUserFromInvitationRequest userInfo)
-            throws InvalidArgumentsException, UserCreationException, UserRetrievalException, EntityRetrievalException,
-            MessagingException, JsonProcessingException, EntityCreationException {
+            throws ValidationException, EntityRetrievalException, InvalidArgumentsException, UserRetrievalException,
+            UserCreationException, MessagingException, JsonProcessingException, EntityCreationException {
 
         if (userInfo.getUser() == null || userInfo.getUser().getSubjectName() == null) {
-            throw new InvalidArgumentsException("Username ('subject name') is required.");
+            throw new ValidationException("Username ('subject name') is required.");
```

Comment:
As part of improving this code, can you move strings like this into our errors file and use ErrorMessageUtil to send them back?

Reply:
Fixed